config/secrets.yml

```diff
@@ -1,5 +1,7 @@
 common: &common
   secret_key_base: 'f0b774b4d06f7ea3357dec260cd6231367009d35525b3c437b27e299d52599bebd4eb1cd83fc3f69ff0d46a0510e5e4ba3bffd26b503b7713100f6083fc4baba'
+  auth_secret_key: '89e115653bae4fafda3fc4c45b64aa07e20ad4d75ab4c1637e85e44b50e4a5cd24ea4dc065e047642d2ed17bebae7e856a292ab1f46ff7d04fc473df1a9ac577'
```

Comment:
I usually don't require everyone to make their own secrets for test/dev. If they are binding to localhost it should never matter. If they aren't binding to localhost then someone who sees their dev/test server and knows about this repo could do various things depending on the secret in question (with `secret_key_base` they could fake sessions) but that seems really low risk.

When including values where there is no test/dev version (such as AWS credentials) usually I mock it out in test so that tests still run and provide instructions in the readme.